tensorrt_llm/_torch/models/modeling_utils.py

```
@@ -679,11 +688,40 @@ def forward(
         )
 
     def load_weights(self,
-                     weights: Dict,
+                     weights: dict[str, torch.Tensor],
                      weight_mapper: Optional["BaseWeightMapper"] = None,
```

Ruff (0.15.20)

[error] Undefined name `BaseWeightMapper`

(F821)